lib/commands.js

```diff
@@ -628,6 +629,45 @@ class Commands {
             inferredConcertoJsonModel.models[0]
         );
     }
+
+    /**
+     * Decorate a given model with given list of dcs and vocab files and print the result
+     * @param {string} modelFile - the model to which vocab and decorator has to be applied
+     * @param {string[]} dcsFiles - the decorator files to be applied to model
+     * @param {string[]} vocFiles - the vocab files to be applied to model
+     * @param {object} options - optional parameters
+     * @returns {string} - Depending on the options, CTO string or JSON string
+     */
+    static async decorate(modelFile, dcsFiles,vocFiles,options) {
+        try {
+            const modelContent = fs.readFileSync(path.resolve(modelFile), 'utf-8');
+            const allDCSFiles = dcsFiles ? dcsFiles.map(file => fs.readFileSync(file, 'utf8')) : [];
+            const allVocsFiles = vocFiles ? vocFiles.map(file => fs.readFileSync(file, 'utf8')) : [];
+            let modelManager = new ModelManager();
+            modelManager.addModel(modelContent);
+            allDCSFiles.forEach(content => {
+                modelManager = DecoratorManager.decorateModels(modelManager, JSON.parse(content));
+            });
+            const vocManager = new VocabularyManager({ missingTermGenerator: VocabularyManager.englishMissingTermGenerator });
+            const namespace = modelManager.getModels().map(obj=>obj.name.replace(/\.cto$/, ''))[0];
+            allVocsFiles.forEach(content => {
+                vocManager.addVocabulary(content);
+            });
+            const vocabKeySet=(Object.keys(vocManager.vocabularies)).map(key=>key.replace(new RegExp(`^${namespace}/`), ''));
+            vocabKeySet.map(voc=>{
+                let commandSet = vocManager.generateDecoratorCommands(modelManager, voc);
+                modelManager = DecoratorManager.decorateModels(modelManager, commandSet);
+            });
+            let modelAst = modelManager.getModelFile(namespace).getAst();
+            let data =  (options.format === 'cto') ? Printer.toCTO(modelAst):JSON.stringify(modelAst);
+            if (options.output) {
+                fs.writeFileSync(options.output, data);
```

Comment:
I'd expect the file name from the original ModelFile to be concatenated with the outputDir, so that a set on input model files get saved into an output directory. E.g. `./decorated/one.cto` and `./decorated/two.cto`.